sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterLoggingExtensions.cs

```diff
@@ -30,6 +30,9 @@ public static OpenTelemetryLoggerOptions AddAzureMonitorLogExporter(this OpenTel
             var options = new AzureMonitorExporterOptions();
             configure?.Invoke(options);
 
+            // Ideally user should set this to true
+            // but if they miss we may have an issue of missing state values which gets converted to custom dimensions.
+            loggerOptions.ParseStateValues = true;
```

Comment:
Consider moving this to before the call to configure. The idea being if the user really wants to turn it off for some reason to avoid parsing state (which does have a little perf hit) they can.